### PR TITLE
[improve][doc] SEO for Tutorials

### PR DESCRIPTION
**Pull request by zuobiao-zhou**
This PR do SEO for Tutorials

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: docs/cookbooks-message-queue.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 id: cookbooks-message-queue
 title: Use Pulsar as a message queue
 sidebar_label: "Message queue"
+description: Learn to use Pulsar as a message queue.
```
Comment (Anonymitaet) [inline docs/cookbooks-message-queue.md:5]:
```suggestion
description: Learn how to use Pulsar as a message queue.
```

File: docs/cookbooks-retention-expiry.md [hunk 2]
```diff
@@ -36,7 +37,7 @@ Pulsar's [admin interface](admin-api-overview.md) enables you to manage both ret
 By default, when a Pulsar message arrives at a broker, the message is stored until it has been acknowledged on all subscriptions, at which point it is marked for deletion. You can override this behavior and retain messages that have already been acknowledged on all subscriptions by setting a *retention policy* for all topics in a given namespace. Retention is based on both a *size limit* and a *time limit*.
 
 The diagram below illustrates the concept of message retention.
-![](/assets/retention.svg)
+![concept of message retention in Pulsar.](/assets/retention.svg)
```
Comment (Anonymitaet) [inline docs/cookbooks-retention-expiry.md:40]:
```suggestion
![Concept of message retention in Pulsar.](/assets/retention.svg)
```

File: docs/cookbooks-retention-expiry.md [hunk 3]
```diff
@@ -188,7 +189,7 @@ admin.namespaces().getRetention(namespace);
 You can control the allowable size and/or time of backlogs, at the namespace level, using *backlog quotas*. Pulsar uses a quota to enforce a hard limit on the logical size of the backlogs in a topic. Backlog quota triggers an alert policy (for example, producer exception) once the quota limit is reached.
 
 The diagram below illustrates the concept of backlog quota.
-![](/assets/backlog-quota.svg)
+![concept of backlog quota i Pulsar](/assets/backlog-quota.svg)
```
Comment (Anonymitaet) [inline docs/cookbooks-retention-expiry.md:192]:
```suggestion
![Concept of backlog quota in Pulsar](/assets/backlog-quota.svg)
```

File: docs/cookbooks-retention-expiry.md [hunk 4]
```diff
@@ -350,7 +351,7 @@ By default, Pulsar stores all unacknowledged messages forever. This can lead to
 The TTL parameter is like a stopwatch attached to each message that defines the amount of time a message is allowed to stay unacknowledged. When the TTL expires, Pulsar automatically moves the message to the acknowledged state (and thus makes it ready for deletion).
 
 The diagram below illustrates the concept of TTL.
-![](/assets/ttl.svg)
+![concept of TTL in Pulsar](/assets/ttl.svg)
```
Comment (Anonymitaet) [inline docs/cookbooks-retention-expiry.md:354]:
```suggestion
![Concept of TTL in Pulsar](/assets/ttl.svg)
```

File: docs/tutorials-namespace.md [hunk 5]
```diff
@@ -2,6 +2,7 @@
 Id: tutorials-namespace
 title: Create a namespace
 sidebar_label: "Create a namespace"
+description: Learn to create and verify a namespace in Pulsar.
```
Comment (Anonymitaet) [inline docs/tutorials-namespace.md:5]:
```suggestion
description: Learn how to create and verify a namespace in Pulsar.

```

File: docs/tutorials-produce-consume.md [hunk 6]
```diff
@@ -2,6 +2,7 @@
 Id: tutorials-produce-consume
 title: Produce and consume messages
 sidebar_label: "Produce and consume messages"
+description: Learn to produce and consume messages in Pulsar.
```
Comment (Anonymitaet) [inline docs/tutorials-produce-consume.md:5]:
```suggestion
description: Learn how to produce and consume messages in Pulsar.

```

File: docs/tutorials-tenant.md [hunk 7]
```diff
@@ -2,14 +2,15 @@
 Id: tutorials-tenant
 title: How to set up a tenant
 sidebar_label: "Set up a tenant"
+description: Learn to set up a tenant in Pulsar.
```
Comment (Anonymitaet) [inline docs/tutorials-tenant.md:5]:
```suggestion
description: Learn how to set up a tenant in Pulsar.

```

File: docs/tutorials-topic.md [hunk 8]
```diff
@@ -2,6 +2,7 @@
 Id: tutorials-topic
 title: How to create a topic
 sidebar_label: "Create a topic"
+description: Learn to create a topic in Pulsar.
```
Comment (Anonymitaet) [inline docs/tutorials-topic.md:5]:
```suggestion
description: Learn how to create a topic in Pulsar.

```

**Comment by zuobiao-zhou** [comment]
fixed @Anonymitaet PTAL.

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**